app/search.py

```diff
@@ -13,12 +13,13 @@ def open_browser(urls):
     for url in urls:
         webbrowser.open_new_tab(url)
 
-def search_my_term(term, count=10):
-    return [url for url in search(term, stop=20)]
+def search_my_term(term, count):
+    return [url for url in search(term, stop=count)]
 
 
 term = sys.argv[1]
 count = sys.argv[2]
```

Comment:
Cleaner practice `count = sys.argv[2]` since we will no longer use `count` as string